no check for libxt-dev when fribidi is enabled

version: current master
environment: debian 10 amd64
procedure: `./autogen.sh && ./configure && make`
error:
```
In file included from BidiJoin.c:31:
fvwmlib.h:9:10: fatal error: X11/Intrinsic.h: No such file or directory
```
workaround: `apt install libxt-dev`
proposed solution: use pkg-config to check for xt if fribidi is enabled.

--- a/configure.ac
+++ b/configure.ac
@@ -388,6 +388,12 @@ AC_SUBST(DOC_SECTIONS)
 AC_SUBST(DOC_SECTIONS_XML)
 AC_SUBST(DOC_SECTIONS_XML_PATH)
 
+PKG_CHECK_MODULES([xt],[xt],
+[],
+[
+ AC_MSG_ERROR([*** libxt not found. ***])
+])
+
 problem_mandoc=""
 AC_CHECK_PROG(XSLTPROC, xsltproc, xsltproc, "")
 

